THIRD_PARTY_NOTICES.md

```diff
@@ -74,6 +74,7 @@ code, the source code can be found at [https://github.com/newrelic/node-newrelic
 **[optionalDependencies](#optionalDependencies)**
 
 * [@contrast/fn-inspect](#contrastfn-inspect)
+* [@newrelic/native-metrics](#newrelicnative-metrics)
```

Comment:
Something is funny here, this isn't a third-party package.

Reply:
It's weird--I didn't make or commit those changes, but they're in my local git history.

Reply:
I'm going to try this again...

Reply:
Committing package.json and package-lock.json is triggering the husky pre-commit hook. That's what's generating those changes.